server/models/Transaction.js

```diff
@@ -320,7 +315,7 @@ function defineModel() {
   };
 
   Transaction.prototype.hasPlatformTip = function () {
-    return this.data?.isFeesOnTop ? true : false;
+    return Boolean(this.data?.isFeesOnTop && this.kind !== TransactionKind.PLATFORM_TIP);
```

Comment:
Not current scope but would be great to convert `isFeesOnTop` to `hasPlatformTip` at some point.